strongbox-security/strongbox-user-management/src/main/java/org/carlspring/strongbox/users/dto/UserRepositoryReadContract.java

```diff
@@ -0,0 +1,16 @@
+package org.carlspring.strongbox.users.dto;
+
+import java.util.Set;
+
+import org.carlspring.strongbox.authorization.dto.PrivelegieReadContract;
+
+public interface UserRepositoryReadContract
+{
+
+    String getRepositoryId();
+
+    Set<? extends PrivelegieReadContract> getRepositoryPrivileges();
+
+    Set<? extends UserPathPrivelegiesReadContract> getPathPrivileges();
+
+}
```

Comment:
Cosmetic: You shold really turn on the checks in Eclipe for new lines at the end of files...

Reply:
done